yt_dlp/extractor/twitter.py

```diff
@@ -211,7 +265,6 @@ class TwitterIE(TwitterBaseIE):
             'duration': 12.922,
             'timestamp': 1442188653,
             'upload_date': '20150913',
-            'age_limit': 18,
```

Comment:
Are u sure this should be removed? Broken test is better than wrong test

Reply:
Yes. This is a duplicate field as @bashonly noticed. A few lines further down is the same field again.